packages/testing-helpers/src/litFixture.js

```diff
@@ -1,25 +1,48 @@
+import { TemplateResult } from 'lit-html';
 import { fixtureWrapper } from './fixtureWrapper.js';
 import { render } from './lit-html.js';
 import { elementUpdated } from './elementUpdated.js';
 
+/**
+ * @typedef {
+     import('lit-html').TemplateResult
+   | import('lit-html').TemplateResult[]
+   | Node
+   | Node[]
+   | string
+   | string[]
+   | number
+   | number[]
+   | boolean
+   | boolean[]
+ } LitHTMLRenderable
+ */
+
+const isCommentNode = ({ nodeType }) => nodeType === 8;
+
+const not = p => x => !p(x);
+
 /**
  * Setups an element synchronously from the provided lit-html template and puts it in the DOM.
  *
  * @template {Element} T - Is an element or a node
- * @param {import('lit-html').TemplateResult} template
+ * @param {LitHTMLRenderable} template
  * @returns {T}
  */
 export function litFixtureSync(template) {
   const wrapper = fixtureWrapper();
   render(template, wrapper);
-  return /** @type {T} */ (wrapper.children[0]);
+  const [node] = Array.from(
+    template instanceof TemplateResult ? wrapper.children : wrapper.childNodes,
```

Comment:
Why not `firstElementChild` ?

Reply:
could be `TextNode`

Reply:
Fair enough, I hadn't really considered that to be a use case.

Reply:
huh...
<img width="941" alt="Screen Shot 2019-10-27 at 10 48 15" src="https://user-images.githubusercontent.com/1466420/67632083-51f8d680-f8a7-11e9-8771-1202bee69aa4.png">

Reply:
Ok I addressed this comment with a small optimization. If it's a `TemplateResult`, we can skip the iteration.